tox.ini

```diff
@@ -0,0 +1,59 @@
+# This configuration file is for Tox. https://tox.readthedocs.io/
+# It houses configuration sections for common Python tools, too.
+
+[tox]
+envlist =
+    flake8
+    pylint
+    readme
+    py{27,33,34}-django17
+    py{27,33,34,35}-django18
+    py{27,34,35}-django19
+    py{27,34,35}-django110
+    py{27,34,35,36}-django111
+    py{34,35,36}-django20
+
+[testenv]
+commands =
+    flake8: flake8
+    pylint: pylint --rcfile=tox.ini pylint_django setup
+    readme: python setup.py check --restructuredtext --strict
+    py{27,34,35,36}-django{17,18,19,110,111,20}: coverage run test/test_func.py
+    clean: find . -type f -name '*.pyc' -delete
+    clean: find . -type d -name __pycache__ -delete
+    clean: rm -rf build/ .cache/ dist/ .eggs/ pylint_django.egg-info/ .tox/
+deps =
+    flake8: flake8
+    pylint: pylint
+    pylint: Django>=2.0,<2.1
+    readme: readme_renderer
+    django17: Django>=1.7,<1.8
+    django18: Django>=1.8,<1.9
+    django19: Django>=1.9,<1.10
+    django110: Django>=1.10,<1.11
+    django111: Django>=1.11,<2.0
+    django20: Django>=2.0,<2.1
+    py{27,34,35,36}: coverage
```

Comment:
You install coverage in the after_success section in .travis.yml, I believe you don't need it here.

Reply:
No, it's [coveralls](https://github.com/PyCQA/pylint-django/pull/126/files#diff-354f30a63fb0907d4ad57269548329e3R39).

Reply:
`coveralls` is installed in the Travis configuration, because it's only needed on Travis.

The `coverage` package is needed for Tox, for both local execution and builds on Travis.